### PR TITLE
Adding image attribution, closes #1537

### PR DESCRIPTION
**Pull request by davetaz**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "adapt-contrib-hotgraphic",
-  "version": "2.0.7",
+  "version": "2.0.8",
```
Comment (moloko) [inline bower.json]: version should actually be 2.0.9 - some idiot (me!) forgot to bump the version when releasing [2.0.8](https://github.com/adaptlearning/adapt-contrib-hotgraphic/releases/tag/v2.0.8)

File: README.md [hunk 2]
```diff
@@ -86,7 +90,7 @@ When viewport size changes to the smallest range, this component will behave lik
 
 
 ----------------------------
-**Version number:**  2.0.7   <a href="https://community.adaptlearning.org/" target="_blank"><img src="https://github.com/adaptlearning/documentation/blob/master/04_wiki_assets/plug-ins/images/adapt-logo-mrgn-lft.jpg" alt="adapt learning logo" align="right"></a> 
+**Version number:**  2.0.8   <a href="https://community.adaptlearning.org/" target="_blank"><img src="https://github.com/adaptlearning/documentation/blob/master/04_wiki_assets/plug-ins/images/adapt-logo-mrgn-lft.jpg" alt="adapt learning logo" align="right"></a>
```
Comment (moloko) [inline README.md]: could you change this to 2.0.9?

Comment (sarveshwar-gavhane) [inline README.md]: is attribution missing here https://github.com/davetaz/adapt-contrib-hotgraphic/blob/e94590a3a1d44ea26cc464408a519a63466be205/example.json#L20-L23

Comment (davetaz) [inline README.md]: yup, sorry. In there now

**Comment by moloko** [comment]
I'll sort the version numbers out when I release

**Pull request merged**